xbmc/interfaces/json-rpc/ApplicationOperations.cpp

```diff
@@ -104,7 +105,7 @@ JSONRPC_STATUS CApplicationOperations::Quit(const std::string &method, ITranspor
 JSONRPC_STATUS CApplicationOperations::GetPropertyValue(const std::string &property, CVariant &result)
 {
   if (property == "volume")
-    result = static_cast<int>(g_application.GetVolumePercent());
+    result = std::lroundf(g_application.GetVolumePercent());
```

Comment:
You may still need to static cast here.

Reply:
Yes, I added `static_cast<int>(...)` to cast `long int` to `int`.